cprover_bindings/src/irep/to_irep.rs

```diff
@@ -254,6 +254,28 @@ impl ToIrep for ExprValue {
                     )],
                 }
             }
+            ExprValue::HalfConstant(i) => {
+                let c: u16 = i.to_bits();
+                Irep {
+                    id: IrepId::Constant,
+                    sub: vec![],
+                    named_sub: linear_map![(
+                        IrepId::Value,
+                        Irep::just_bitpattern_id(c, mm.float_width, false)
```

Comment:
Should the width be 16 instead of `mm.float_width`?

Reply:
Yes, thank you